packages/patterns-react/src/patterns/PictogramArray/README.md

```diff
@@ -73,6 +73,27 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 > 💡 Don't forget to import the leadspace styles from
 > [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
 
+#### Markdown
+
+Some markdown syntaxes are accepted within the copy of the content item
+
+Example:
+
+```javascript
+{
+  copy: '_Lorem_ ipsum __dolor__ sit amet.',
+  ...
+}
+```
+
+Renders:
+
+_Lorem_ ipsum **dolor** sit amet.
```

Comment:
All we need is italic. Bold isn't required and is in fact deeply frowned upon by the web simplification team. It denotes/changes content hierarchy they said.

Reply:
Oh right, I forgot. Fixing...

Reply:
Fixed